devices.js

```diff
@@ -1383,6 +1383,15 @@ const devices = [
     },
 
     // TuYa
+    {
```

Comment:
Move this entry to the bottom of the file (together with other silvercrest devices)

Reply:
Done